controller/comments.controller.js

```diff
@@ -1,18 +1,25 @@
-const { selectComments } = require('../model/comments.model')
+const { selectArticleId } = require('../model/articles.model');
+const { selectComments, insertComment } = require('../model/comments.model')
 
 const getComments = (req, res, next) => {
     const article_id = req.params.article_id;
-    selectComments(article_id)
-    .then((comments) => {
-        if (comments && comments.length > 0) {
-            res.status(200).send(comments)
-        } else {
-            res.status(404).send({status: 404, msg: 'Unable to find comments'})
-        }
-    })
-    .catch((error) => {
+    selectArticleId(article_id).then(() => {
+        return selectComments(article_id)
+    }).then((comments) => {
+        res.status(200).send(comments)
+    }).catch((error) => {
         next(error)
     })
     }
 
-module.exports = { getComments }
\ No newline at end of file
+
+const postComment = (req, res, next) => {
+    const articleID = req.params.article_id;
+    selectArticleId(articleID).then(() => insertComment(articleID, req.body)
+    ).then((comment) => 
+    res.status(201).send(comment)
```

Comment:
Remember to send the comment back on a key of comment not just as the body itself - `send({comment})`. You will need to amend your tests first